### DISCUSSION TITLE
Advanced management of Wildcards collections.

### DISCUSSION THREAD

**Discussion by DAYNEMOS** [adieyal/sd-dynamic-prompts#735] [Ideas]
How possible is it to add a built-in function for creating/deleting .txt Wildcards files, without having to open the Wildcards directory every time, and do everything outside the webui working environment? I mean... Adding functional buttons "Create new Wildcards", as well as fields for entering the name of the new Wildcards, and the path to save the new Wildcards (or the standard path in the wildcards root folder).

**Comment by rpillala** [comment]
Even a button on the page to open the wildcards location would be very convenient for me. I'm no programmer so I don't know how difficult this is. 

Btw this addon is one of the only reasons I'm still using stable diffusion. I'm just a hobbyist, not doing any commercial use or anything, and without prompt variety, SD would have gotten boring a long time ago.
